bundles/org.openhab.ui.homebuilder/web/i18n/vi-VN.json

```diff
@@ -0,0 +1,62 @@
+{
+    "GroundFloor": "",
```

Comment:
@ThomDietrich This does not look okay to me could we leave out files without translations?

Reply:
Sadly not possible, see https://github.com/openhab/openhab-core/pull/244#issuecomment-349934843
Not ideal but @kubawolanin already confirmed that he'll take care of this in the HomeBuilder code.